messages/culture.txt

```diff
@@ -31,4 +31,17 @@
 <b>It is normal to hate your job once in a while</b> It is ok to hate your job sometimes. In fact, if you hate your job once in a while, it means you are probably growing. I have hated my job at Qxf2 a few times in the past. I even hated playing chess on some days when I was still competing. This is normal. You do not have to feel guilty about it. If you are disliking your job for several weeks together, please talk to your 1:1 manager. They will try to help you decide on what to change. Almost always, changing the nature of work will not be the answer. Instead, it will probably be changing mindset and/or your outlook that helps.
 <b>Find your baseline</b> Many professionals have made no attempt at figuring out how different setups, work sequences, constraints, and distractions influence their work output. Treat this as a testing problem. Experiment. Vary different portions of your work and see what suits you. The productivity gains will be good but that is a secondary concern. The primary benefit I got by doing this exercise was to recognize the many, many invisible factors that influence work.
-<b>Honest retrospectives</b> Please make an effort to fill out your retrospective well. Good, honest, open and thoughtful retrospectives serve as a culture reinforcement tool. Good retrospectives help normalize the fact that all of us are struggling and doing good at the same time. This will help the new hire identify with us quicker. Open retrospectives remove the stigma around failure. The more honestly we share our experience, the more likely that the new folks will trust us. The more thoughtful our retrospectives, the more likely that our new hires will be more introspective.
\ No newline at end of file
+<b>Honest retrospectives</b> Please make an effort to fill out your retrospective well. Good, honest, open and thoughtful retrospectives serve as a culture reinforcement tool. Good retrospectives help normalize the fact that all of us are struggling and doing good at the same time. This will help the new hire identify with us quicker. Open retrospectives remove the stigma around failure. The more honestly we share our experience, the more likely that the new folks will trust us. The more thoughtful our retrospectives, the more likely that our new hires will be more introspective.
```

Comment:
Can we reframe this sentence "The more honestly we share our experience, the more likely that the new folks will trust us. The more thoughtful our retrospectives, the more likely that our new hires will be more introspective."
 to 
"
The more honestly we share our experiences, the more likely it is that the new folks will trust us. The more thoughtful our retrospectives are, the more likely it will be that the new hires will do their introspective.
"

Reply:
@rohinigopalqxf2 - I will avoid this change because I am not sure if the change is correct English or not. Grammarly is ok with it. But when I googled for "will do their introspective" (in quotes), I got zero results. So I suspect ti is better to leave it like this.